### PR TITLE
fix: BFF-1331 Removed user presets from user ext and added into a separate…

### PR DESCRIPTION
**Pull request by alexbejann**
… class based on guid

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/features/amount_presets/models/user_presets.dart [hunk 1]
```diff
@@ -1,8 +1,34 @@
 import 'package:equatable/equatable.dart';
 import 'package:givt_app/features/amount_presets/models/preset.dart';
 
+class AmountPresets extends Equatable {
```
Comment (pavelpavelko) [inline lib/features/amount_presets/models/user_presets.dart:4]: would be better to move it in a separate file amount_presets.dart?

File: lib/features/first_use/pages/welcome_page.dart [hunk 2]
```diff
@@ -78,7 +78,8 @@ class _WelcomePageViewState extends State<WelcomePageView> {
               Padding(
                 padding: const EdgeInsets.symmetric(vertical: 15),
                 child: ElevatedButton(
-                  onPressed: () => Navigator.of(context).push(EmailSignupPage.route()),
+                  onPressed: () =>
+                      Navigator.of(context).push(EmailSignupPage.route()),
```
Comment (pavelpavelko) [inline lib/features/first_use/pages/welcome_page.dart:82]: why native navigator instead of go_router approach is used here?

**Review by pavelpavelko — APPROVED** [review]
lgtm

File: lib/features/amount_presets/models/user_presets.dart (hunk 1 above)
Comment (alexbejann) [reply lib/features/amount_presets/models/user_presets.dart:4]: Yes, you're right!

**Pull request merged**